src/notebook.h

```diff
@@ -197,19 +197,33 @@ class MKCAL_EXPORT Notebook
     void setIsReadOnly(bool isReadOnly);
 
     /**
-      Returns true if notebook is visible.
+      Returns true if incidence of this notebook are visible.
       @see setIsVisible().
     */
     bool isVisible() const;
 
     /**
-      Set notebook visibility.
+      Set notebook incidence visibility.
       Calendar will check this value for including/excluding incidences
       into search lists.
       @param isVisible true to set visible mode.
     */
     void setIsVisible(bool isVisible);
 
+    /**
+      Returns true if this notebook is enabled (disabled calendars are not shown to the user).
+      @see setIsEnabled().
+    */
+    bool isEnabled() const;
+
+    /**
+      Set if notebook is enabled.
+      Disabled calendars are not shown to the user, are not synced... They are still
+      present in the database though.
```

Comment:
These comments could be slightly adjusted. We don't know if the notebooks are synced or not and shouldn't prohibit that. Also can't say for sure what are shown to user so maybe rather suggest "disabled notebooks shouldn't be shown to user".